pulser-core/pulser/json/exceptions.py

```diff
@@ -11,26 +11,24 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Custom exceptions for serialization errors."""
+"""Custom exceptions for serialization errors.
 
+Note: This module has migrated to pulser.exceptions.serialization. Please
+use the new path for any future work.
+"""
 
-class SerializationError(Exception):
-    """Exception raised when sequence serialization fails."""
+import warnings
 
-    pass
+from pulser.exceptions import serialization
 
+warnings.warn(
+    "module pulser.json.exceptions is deprecated, "
+    "please migrate your code to "
+    "use pulser.exceptions.serialization"
+)
```

Comment:
```suggestion
warnings.warn(
    "module pulser.json.exceptions is deprecated, "
    "please migrate your code to "
    "use pulser.exceptions.serialization", 
    category=DeprecationWarning
)
```
You can also try to adjust the `stacklevel` I'm not sure what's the best value in this case

Reply:
Good catch. At least testing on the REPL, `stacklevel=2` seems to be the only one to actually display a warning, so I'm putting that.